Sources/ButtonBarViewCell.swift

```diff
@@ -26,19 +26,24 @@ import Foundation
 
 open class ButtonBarViewCell: UICollectionViewCell {
     
-    @IBOutlet open var imageView: UIImageView!
-    @IBOutlet open lazy var label: UILabel! = { [unowned self] in
-        let label = UILabel(frame: self.contentView.bounds)
-        label.autoresizingMask = [.flexibleWidth, .flexibleHeight]
-        label.textAlignment = .center
-        label.font = UIFont.boldSystemFont(ofSize: 14.0)
-        return label
-    }()
+    @IBOutlet weak open var imageView: UIImageView!
+    @IBOutlet weak open var label: UILabel!
+    
+    open override func awakeFromNib() {
+        super.awakeFromNib()
+        
+        let labelAux = label ?? UILabel()
+        labelAux.frame = self.contentView.bounds
+        labelAux.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        labelAux.textAlignment = .center
+        labelAux.font = UIFont.boldSystemFont(ofSize: 14.0)
```

Comment:
If you do not create a label in your xib then the 'lazy body' will create the label for you. In fact you might also use this class without a nib file and in that case you must create the label at some point. 
So I think the label should be created in the `init(frame:)` method as that covers the case of not using a nib file. In addition it should be created in `awakeFromNib` if it is nil.